### PR TITLE
Issue #254: Declared media query for desktop inside <link>

### PR DESCRIPTION
**Pull request by abhishekvp**

### CODE REVIEW

File: index.html~ [hunk 1]
```diff
@@ -0,0 +1,656 @@
+<!DOCTYPE html>
```
Comment (ferndot) [inline index.html~]: Hmm, it looks like your editor generated this file. Could you remove it?

File: style/desktop.css~ [hunk 2]
```diff
@@ -0,0 +1,195 @@
+@media (min-width: 767px) {
```
Comment (ferndot) [inline style/desktop.css~]: It generated this file too.

File: style/desktop.css [hunk 3]
```diff
@@ -1,4 +1,3 @@
-@media (min-width: 767px) {
   /* Regions */
```
Comment (ferndot) [inline style/desktop.css]: Could you decrease the indentation of the styles by two spaces? Thanks!

**Comment by ferndot** [comment]
Thank you for this pull request @abhishekvp! I have made a few comments in the diff. You can update the pull request by pushing new commits to your branch.

**Comment by twiss** [comment]
:+1:

**Comment by ferndot** [comment]
:+1: this is ready for merge!

@abhishekvp if you would be willing, I would greatly appreciate if you squash your changes. You can do so by running:
`$ git rebase -i develop`
VIM or another command line editor will appear. Replace the `pick` next to the title of your second commit with `squash`. Then, save the file. Finally, run:
`$ git push origin issue-254 --force`

**Comment by Ryuno-Ki** [comment]
Oh, and leave the uppermost commit as is.
Otherwise you will notice no difference (this happened to me the first time).

squashing commits means, that all the commits get summed up into one. This way, the history becomes cleaner.

**Comment by ferndot** [comment]
Yep. @abhishekvp if you would rather that I did it, that is fine.

**Comment by abhishekvp** [comment]
@joshua-s @Ryuno-Ki Thanks for your guidance! :) @twiss Thanks! :) I have squashed my commits as suggested.

**Comment by ferndot** [comment]
Merged :smile:

**Pull request merged**